### PR TITLE
Be consistent about casing in Query History menu

### PR DESCRIPTION
**Pull request by elenatanasoiu**
Reported here: https://github.com/github/code-scanning/issues/6008

We originally started out by capitalizing each word [1], but made some
small changes [2] which resulted in our Query History options
being inconsistent.

Let's fix that.

[1]: https://github.com/github/vscode-codeql/blame/a5da5564969a0f37f95f1fe8b856f60810fc5caa/extensions/ql-vscode/package.json
[1]: https://github.com/github/vscode-codeql/commit/b470e4143126d0ceb65aee00fdb25e135deba328

## Questions

Building the extension: I've followed the [contribution docs](https://github.com/github/vscode-codeql/blob/main/CONTRIBUTING.md#building-the-extension) to do this using `npm build`. However, I could only complete the build successfully once I downgraded to node v16.14.2. For node v18 the build fails. Would we be okay with adding a `.nvmrc` file in this repo? 

To provide a bit more context: I only found the correct version by spinning up a codespace and snooping around. "Why don't you build the extension in a codespace?" you ask. While it does build successfully, it doesn't seem like you're able to then install the resulting .vsix file into VSCode. Happy to be told otherwise.

TL;DR: are we comfortable with pinning node to v16.14.2 in an `.nvmrc` file?

## Checklist

- [ ] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [ ] Issues have been created for any UI or other user-facing changes made by this pull request.
- [ ] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

File: extensions/ql-vscode/package.json [hunk 1]
```diff
@@ -471,7 +471,7 @@
       },
       {
         "command": "codeQLQueryHistory.openQuery",
-        "title": "Open the query that produced these results",
+        "title": "Open The Query That Produced These Results",
```
Comment (aeisenberg) [inline extensions/ql-vscode/package.json:474]: minor: Generally, small words shouldn't be capitalized unless they are the first in the sentence.

```suggestion
        "title": "Open the Query that Produced these Results",
```

**Review by aeisenberg — COMMENTED** [review]
Thanks for the PR!

**Comment by aeisenberg** [comment]
Hmmm...I wasn't aware that the extension wasn't building in node v18. It's definitely worth fixing eventually, but in general, we need to stay in sync with the node version shipped with vscode (currently v16.13.0).

I think it is fine to add a `.nvmrc` file. Also, we should update the `engine` field in `package.json` to include a max version as well as min. Using `^16.11.25` should be fine since it allows for anything less than `17.0.0`.

Would you be able to raise an issue for the broken build on v18? It's something we should fix before vscode moves to v18.

**Review by aeisenberg — APPROVED** [review]

**Comment by charisk** [comment]
Seems a bit silly but do we need to update the CHANGELOG file? The PR description check-list suggests yes. cc @shati-patel

**Comment by shati-patel** [comment]
> Seems a bit silly but do we need to update the CHANGELOG file? The PR description check-list suggests yes. cc @shati-patel

No harm in updating the changelog, but it's definitely not necessary in this case since it's such a minor change!

**Comment by elenatanasoiu** [comment]
Thanks @charisk & @shati-patel. Have added a line to the changelog.

**Review by shati-patel — APPROVED** [review]
:shipit: ✅
